lib/tasks/spam_report.rake

```diff
@@ -0,0 +1,68 @@
+namespace :spam do
+
+  desc "Print list of potential spammers"
+  task(:print_possible => :environment) do
+    period = 1.day.ago
+    history_period = 14.day.ago
+    threshold = ArchiveConfig.SPAM_THRESHOLD
+    spam_score = Hash.new
+    spam_works = Hash.new
+    new_works = Work.where("created_at > :week and hidden_by_admin = 'false' ", {:week => period})
+    pseud_list = new_works.collect { |w|  unless  w.pseuds.first.nil? 
```

Comment:
It's better to give things accurate names - "pseud_list" here is actually a list of user ids, so "user_ids" would be clearer and make more sense. Also, it's better practice to use the do/end form for blocks when they cover more than one line.